pcdsdevices/lens.py

```diff
@@ -41,3 +42,12 @@ def __init__(self, x_prefix, y_prefix, z_prefix, *args, **kwargs):
         self.y_prefix = y_prefix
         self.z_prefix = z_prefix
         super().__init__(x_prefix, *args, **kwargs)
+
+
+class SimLensStack(LensStack):
```

Comment:
Still thinking about where these all belong. It would be nice if all these live under the "`.sim`" module but would like to avoid repeating the file structure. Maybe we can think of something clever.

Reply:
I keep flip-flopping. I started to put them into `sim`, but then I thought that everything will be easier if `sim.py` simply provided utilities for creating simulated devices, and then in the device-specific files we put the actual simulated devices. Then the `SimLensStack` is bundled with the `LensStack` and it's really easy to find it.